crates/adkg-cli/README.md

```diff
@@ -105,8 +105,44 @@ adkg-cli run                                \
   --priv adkg.priv                          \
   --id 1                                    \
   --listen-address "/ip4/0.0.0.0/tcp/7777"  \
+  --transcript-out adkg.transcript          \
```

Comment:
not for this PR, but we could consider taking an output folder and storing all the output in set filenames à la drand